src/test/java/com/mentoring/model/User.java

```diff
@@ -60,7 +66,69 @@ private void registerUser(String username, String password, String email) {
         log.info("User registered:\n\tUsername: {}\n\tEmail: {}\n\tPassword: {}", username, email, password);
     }
 
+    public void loginUser(String userName, String password) {
+        Map<String, String> loginData = new HashMap<>();
+        loginData.put("userName", userName);
+        loginData.put("password", password);
+
+        RestAssured.baseURI = Config.getBaseUrl();
+        given()
+                .urlEncodingEnabled(true)
+                .contentType(ContentType.JSON)
+                .body(loginData)
+                .when()
+                .post(getLoginPath())
+                .then()
+                .statusCode(200);
+        log.info("User logged in:\n\tUsername: {}\n\tPassword: {}", userName, password);
+    }
+
+    public void createCharacter(String characterName, String accessToken, String userId) {
+        Map<String, String> characterData = new HashMap<>();
+        characterData.put("characterName", characterName);
+
+        RestAssured.baseURI = Config.getBaseUrl();
+        given()
+                .urlEncodingEnabled(true)
+                .contentType(ContentType.JSON)
+                .body(characterData)
+                .cookie("accesstokenid", accessToken)
+                .cookie("userid", userId)
+                .when()
+                .post(getCharacterPath())
+                .then()
+                .statusCode(200);
+        log.info("User character created:\n\tCharacter name: {}", characterName);
+    }
+
+    public void logoutUser(String userName, String accessToken, String userId) {
```

Comment:
I don't believe you need this. You can simply switch cookies if you want to switch between two authenticated users and you don't need to log the users out after test methods, since users are created independently per test and this class also gets instantiated per test.